specification/maps/resource-manager/Microsoft.Maps/stable/2021-02-01/examples/CreateMapsCreator.json

```diff
@@ -0,0 +1,35 @@
+{
+  "parameters": {
+    "api-version": "2021-02-01",
+    "subscriptionId": "21a9967a-e8a9-4656-a70b-96ff1c4d05a0",
+    "resourceGroupName": "myResourceGroup",
+    "accountName": "myMapsAccount",
+    "creatorName": "myCreator",
+    "CreatorResource": {
+      "location": "unitedstates",
```

Comment:
this is not a valid arm region, please use eastus or similar

Reply:
Creator resource doesn't support valid ARM regions. It supports "unitedstates" and "europe". I understand Azure resource should support valid ARM regions however creator is considered "geographic" and not "regional" so a decision was made to support non-standard ARM regions in location. Let follow up on this.

Reply:
Is there any issue with this at the moment? We will likely need an exception, until the team can move to standard ARM regions.

Reply:
I don't think routing will work in ARM for regions that ARM does not support. Since 'location' has special meaning at the top-level in a resource you are restricted to what ARM supports. The alternative would be to model this as a proxy resource with a customLocation property within the 'properties' section

Reply:
see [https://armwiki.azurewebsites.net/rp_onboarding/tracked_vs_proxy_resources.html?q=tracked](https://armwiki.azurewebsites.net/rp_onboarding/tracked_vs_proxy_resources.html?q=tracked)

Reply:
See our existing prod manifest for [Microsoft.Maps/accounts/privateAtlases](https://armmanifest.visualstudio.com/One/_git/Manifest?path=%2FMICROSOFT.MAPS%2FPROD%2FMANIFEST.JSON&version=GBmaster&line=227&lineEnd=228&lineStartColumn=13&lineEndColumn=15&lineStyle=plain&_a=contents) and [Microsoft.Maps/accounts](https://armmanifest.visualstudio.com/One/_git/Manifest?path=%2FMICROSOFT.MAPS%2FPROD%2FMANIFEST.JSON&version=GBmaster&line=66&lineEnd=67&lineStartColumn=13&lineEndColumn=15&lineStyle=plain&_a=contents). 

Note 
- We have routing working however it results in single wildcard `""` and `"United States"` as location, even our top level resource is has historically been `"Global"` and we are progressing to ARM regions. 
- Creators type will follow the same pattern but later. Switching to proxy type doesn't address the problem because we want to leverage the support of a tracked resource. 
- in other comments, I mentioned "Creators" resource type is **replacing** the deprecated "privateAtlases" type.

As referenced, manifest routing by "united states" **works** but there are known limitations / constraints at the moment including managed identity (`identity` property) which relies on standard ARM regions.

Is this a blocker?